debian/control

```diff
@@ -16,6 +16,7 @@ Build-Depends:
 	libboost-filesystem-dev,
 	libidn11-dev,
 	blackhole-dev (>= 0.2.2-1),
+	virtualenv,
```

Comment:
Does this package exist in other debian/ubuntu versions?
It should probably be:
python-virtualenv | virtualenv